pkg/mutation/logging.go

```diff
@@ -33,9 +33,21 @@ func logAppliedMutations(message string, mutationUUID uuid.UUID, obj *unstructur
 			logging.ResourceKind, obj.GroupVersionKind().Kind,
 			logging.ResourceAPIVersion, obj.GroupVersionKind().Version,
 			logging.ResourceNamespace, obj.GetNamespace(),
-			logging.ResourceName, obj.GetName(),
+			logging.ResourceName, getNameOrGenerateName(obj),
```

Comment:
Should we log `resource_label` as well just like audit logs? if `resource_name` includes generated name and not the actual resource name in such cases, providing `labels` might be helpful.

Reply:
@maxsmythe ok to add labels here?

Reply:
added resource labels for now, easy to revert commit if objections arise